examples/annotation_import/image.ipynb

```diff
@@ -1,18 +1,16 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.          "name": " ",
</pre><p>This should be "mask" not "&lt;insert segementation mask name as seen in ontology&gt; ",</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Done, I was trying to add instructions for users instead of making it demo-able